connection/connection.go

```diff
@@ -0,0 +1,69 @@
+package connection
+
+import (
+	"context"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-csi/csi-lib-utils/protosanitizer"
+	"google.golang.org/grpc"
+)
+
+const (
+	// Interval of logging connection errors
+	connectionLoggingInterval = 10 * time.Second
+)
+
+// Connect opens insecure gRPC connection to a CSI driver. Address must have either '<protocol>://' prefix, or be
```

Comment:
Better document what protocols are valid or link to documentation about that. I know of that people have been bitten by attempts to use "tcp4" as prefix, which is not what gRPC uses (it wants `dns`).

Reply:
Added link to https://github.com/grpc/grpc/blob/master/doc/naming.md